src/utils/legacyLayerParamUtils.js

```diff
@@ -171,3 +171,22 @@ export async function getKmlLayerFromLegacyAdminIdParam(adminId) {
         kmlMetaData
     )
 }
+
+export function handleBodLayerIdParam(params, store, newQuery) {
```

Comment:
I think we should rename/rephrase this param, BOD layer ID isn't much self-explanatory.
What about : `handleLayerFeaturePreSelection`